src/extensions/core/invertMenuScrolling.ts

```diff
@@ -10,17 +9,22 @@ app.registerExtension({
   init() {
     const ctxMenu = LiteGraph.ContextMenu
     const replace = () => {
-      // @ts-expect-error
-      LiteGraph.ContextMenu = function (values, options) {
-        options = options || {}
-        if (options.scroll_speed) {
-          options.scroll_speed *= -1
-        } else {
-          options.scroll_speed = -0.1
+      type CtxMenuConstructorArgs = ConstructorParameters<typeof ctxMenu>
+      class InvertContextMenu extends ctxMenu {
+        constructor(
```

Comment:
Replace "function + function.prototype" with a extended class that overwrites the constructor with "the function", to get rid of @ts-expect-error.